### PR TITLE
Release 2.1.1

### PR DESCRIPTION
**Pull request by ialokim**
This PR prepares the next release `2.1.1`.

* [x] update PTE to the latest commit at [opentransitmap](https://gitlab.com/opentransitmap/public-transport-enabler/-/commits/master) and run `./update-dependency-pinning.sh`
* [x] test the app (automatically using AndroidTest as well as manually), paying special attention to changes since the last release
* [x] fix any bugs if necessary
* [x] update translations from [Transifex](https://www.transifex.com/) using `tx pull --mode=developer -a` in the root folder (you need proper permissions to do that)
* [x] add newly supported languages to the two arrays in [`app/src/main/res/values/arrays.xml`](https://github.com/grote/Transportr/blob/master/app/src/main/res/values/arrays.xml#L16)
* [x] revise the commits to `master` since the last release and add interesting changes (as well as new languages) to the changelog at [`app/src/main/res/xml/changelog_master.xml`](https://github.com/grote/Transportr/blob/master/app/src/main/res/xml/changelog_master.xml), then run `python3 ./fastlane/generate_changelog.py`
* [x] bump the [`versionCode`](https://github.com/grote/Transportr/blob/master/app/build.gradle#L14) and [`versionName`](https://github.com/grote/Transportr/blob/master/app/build.gradle#L15) in `app/build.gradle`
* [x] add a last commit updating all dependencies right after the release

### CODE REVIEW

**Comment by ialokim** [comment]
This should be ready now. The release tag should be added to the `release 2.1.1` commit, not the last one.

File: app/build.gradle [hunk 1]
```diff
@@ -117,11 +117,11 @@ dependencies {
 	implementation 'androidx.cardview:cardview:1.0.0'
 	implementation 'androidx.recyclerview:recyclerview:1.1.0'
 	implementation 'androidx.legacy:legacy-preference-v14:1.0.0'
-	implementation 'androidx.constraintlayout:constraintlayout:2.0.1'
+	implementation 'androidx.constraintlayout:constraintlayout:2.0.4'
```
Comment (grote) [inline app/build.gradle:120]: Updating libraries for a release?

Comment (ialokim) [reply app/build.gradle:120]: This is just contained in the last, follow-up commit. The release should be at commit [`release 2.1.1`](https://github.com/grote/Transportr/pull/736/commits/d76cb30b96bda0e83209076ac645a0e61e080485).

Comment (grote) [reply app/build.gradle:120]: Ah, I would prefer follow-up stuff in a dedicated PR.

**Comment by grote** [comment]
Thanks a lot for all your work! The release has been published. In Google Play it is in the beta track. If there's no new issues popping up, we can promote it to production soon.

**Pull request merged**